sdk/tables/azure-data-tables/inc/azure/data/tables/table_client.hpp

```diff
@@ -50,6 +51,12 @@ namespace Azure { namespace Data { namespace Tables {
      * to prompt a challenge in order to discover the correct tenant for the resource.
      */
     bool EnableTenantDiscovery = false;
+
+    /**
+     * The Audience to use for authentication with Azure Active Directory (AAD).
+     * Audience will be assumed based on serviceUrl if it is not set.
+     */
+    Azure::Nullable<TableAudience> Audience;
```

Comment:
Is there only a single audience? The Authorization logic take a vector of authorization scopes, does it make sense to replicate that here?

Reply:
Also, to Anton's comment: Do we want to add this to azure_core even if it's not respected in existing service clients?

Reply:
I noticed the authorization vector, since before it was only one, and storage is also only one thing, same as others, also to avois confusiopn for customers, i stuck to the previous model. 
As for the adding it to azure core, i was debating it since everyone has audience thing , and it would help centralize things, i'm not sure the benefit right now for customers, and unless we come up with a list or scopes it would be kind of half work done, without any real benefit for customers. I'd rather do it when we figure out and spend some thime figuring out these scopes in a more general fashion for all the services.